app/__init__.py

```diff
@@ -10,15 +10,25 @@ def create_app():
 
     app.config['SECRET_KEY'] = 'secret-key-goes-here'
     app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.sqlite'
+    # adding to suppress warning, will delete later
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
 
     db.init_app(app)
+
     
     login_manager = LoginManager()
     login_manager.login_view = 'auth.login'
     login_manager.init_app(app)
 
     from .models import User
 
+    ######
+    # logan kiser: troubleshooting db issues, will delete before submitting
+    #              pull request.
+    # with app.app_context():
+        # db.create_all()
+    ######
```

Comment:
I'll touch base with others to find a better way of doing this. Most likely pushing a context via command line.